data/src/test/java/org/apache/iceberg/data/TestLocalScan.java

```diff
@@ -77,19 +76,11 @@ public class TestLocalScan {
   private static final Configuration CONF = new Configuration();
   private static final Tables TABLES = new HadoopTables(CONF);
 
-  @Rule public final TemporaryFolder temp = new TemporaryFolder();
-
-  @Parameterized.Parameters(name = "format = {0}")
-  public static Object[] parameters() {
```

Comment:
no need to rename this and change the return type. This is also missing `@Parameters` annotation